burn-import/src/onnx/op_configuration.rs

```diff
@@ -123,6 +121,42 @@ pub fn max_pool2d_config(curr: &Node) -> MaxPool2dConfig {
         .with_dilation([dilations[0] as usize, dilations[1] as usize])
 }
 
+#[rustfmt::skip]
+pub fn conv_transpose2d_config(curr: &Node) -> ConvTranspose2dConfig {
+    let mut attrs = curr.attrs.clone();
+    let kernel_shape = attrs.remove("kernel_shape").map(AttributeValue::into_i64s).unwrap_or_default();
+    let stride = attrs.remove("strides").map(AttributeValue::into_i64s).unwrap_or_else(|| vec![1, 1]);
+    let pads = attrs.remove("pads").map(AttributeValue::into_i64s).unwrap_or_else(|| vec![0, 0]);
+    let dilations = attrs.remove("dilations").map(AttributeValue::into_i64s).unwrap_or_else(|| vec![1, 1]);
+    let group = attrs.remove("group").map(AttributeValue::into_i64).unwrap_or(1);
+
+    // Trick with remove + empty check is simplest way to not forget some attribute for runtime:
+    if !attrs.is_empty() {
```

Comment:
IMO, this is much more safe than omitting unknown parameters. I would like to discuss the pros and cons of this solution  😃

Reply:
Yes. Probably this can safeguard for cases when not accounted. However, looping through items seems cleaner than `.remove`. Maybe there is a way to pop an item and check remaining?